tests/operators/test_autodiff.py

```diff
@@ -75,15 +75,14 @@ def test_adjoint_and_grad(operator, interface):
     if operator.backend == "finufft" and "gpu" in interface:
         pytest.skip("GPU not supported for finufft backend")
 
-    if torch.is_tensor(operator.samples):
-        operator.samples = operator.samples.cpu().detach().numpy()
-
-    operator.samples = to_interface(operator.samples, interface=interface)
+    if "gpu" in interface:
+        operator.to("cuda")
+    else:
+        operator.cpu()
```

Comment:
what does this mean ? operator is not a tensor here.

Reply:
`operator` is a `torch.nn.module` here. This means all the underlying parameters / weights will be on GPU or CPU. It is very "torch-like" code, and this is good for us